low-level/json_msgs/schemas/actuators/SSPL-LL_Actuator_Response.json

```diff
@@ -119,22 +119,6 @@
 								}
 							}
 						},
-
```

Comment:
what about adding new schema addition here for validation?

Reply:
Previously "service_controller" schema was not align with other actuator schema hence we  separate schema was there for this actuator, now as we have align this actuator schema with other actuator , no need to add separate schema it will get validated with common schema.(line 122-212)

Reply:
Ok